install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -31,6 +31,24 @@ data:
     # Set accessLogEncoding to JSON or TEXT to configure sidecar access log
     accessLogEncoding: '{{ .Values.global.proxy.accessLogEncoding }}'
 
+    {{- if and .Values.global.istioRemote }}
+    
+    {{- if .Values.global.remotePolicyAddress }}
+    {{- if .Values.global.createRemoteSvcEndpoints }}
+    mixerCheckServer: istio-policy.{{ .Release.Namespace }}:15004
+    {{- else }}
+    mixerCheckServer: {{ .Values.global.remotePolicyAddress }}:15004
+    {{- end }}
+    {{- end }}
+    {{- if .Values.global.remoteTelemetryAddress }}
+    {{- if .Values.global.createRemoteSvcEndpoints }}
+    mixerReportServer: istio-telemetry.{{ .Release.Namespace }}:15004
+    {{- else }}
+    mixerReportServer: {{ .Values.global.remoteTelemetryAddress }}:15004
+    {{- end }}
+    {{- end }}
```

Comment:
I copy-pasted it from istio-remote's configmap.
Which variables are you referring to?

Reply:
That makes sense.

I am not clear the istio-remote configmap is correct - and don't want to crater the existing configmap with regressions.

There was a significant divergence between 1.0 and 1.1 configmap, and the istio-remote configmap was not maintained in the process Complicating matters is the fact that 1.1 (or 1.0?) uses EDS for telemetry IIRC.

Reply:
We'll need to test it, hard to tell by looking. We'll probably iterate a bit - with the new installer setting the address of each component will become common since istio-system will likely go away.